sdk/java-sdk-protobuf/src/main/scala/kalix/javasdk/impl/DiscoveryImpl.scala

```diff
@@ -76,7 +76,7 @@ class DiscoveryImpl(
    */
   override def discover(in: ProxyInfo): scala.concurrent.Future[Spec] = {
     log.info(
-      "Received discovery call from [{} {}] at [{}]:[{}] supporting Kalix protocol {}.{}",
+      "ServiceInfo [{} {}] at [{}]:[{}] supporting Kalix protocol {}.{}",
```

Comment:
The previous version was more informative to me.

Reply:
It is also now for me. Changed back.